### PR TITLE
Update latest weave net

### PR DESCRIPTION
**Pull request by dims**
Looking to see if 2.0.5 fixes problems in kubeadm tests:
https://github.com/kubernetes/kubernetes/issues/53570#issuecomment-335582560

### CODE REVIEW

File: phase3/weave-net/weave-net-daemon-set.json [hunk 1]
```diff
@@ -1,133 +1,163 @@
...
-                        "hostPath": {
-                            "path": "/lib/modules"
-                        }
-                    }
-                ]
-            }
-        }
-    }
-}
+	"apiVersion": "extensions/v1beta1",
```
Comment (luxas) [inline phase3/weave-net/weave-net-daemon-set.json]: apps/v1beta2 now...

File: phase3/weave-net/weave-net-daemon-set.json [hunk 2]
```diff
@@ -1,133 +1,163 @@
...
+		"labels": {
+			"name": "weave-net"
+		},
+		"namespace": "kube-system"
+	},
+	"spec": {
+		"template": {
+			"metadata": {
+				"annotations": {
+					"scheduler.alpha.kubernetes.io/tolerations": "[{\"key\":\"dedicated\",\"operator\":\"Equal\",\"value\":\"master\",\"effect\":\"NoSchedule\"}]"
```
Comment (luxas) [inline phase3/weave-net/weave-net-daemon-set.json]: This annotation is not respected and should not be used

File: phase3/weave-net/weave-net-daemon-set.json [hunk 3]
```diff
@@ -1,133 +1,163 @@
...
+				],
+				"restartPolicy": "Always",
+				"tolerations": [
+					{
+						"key": "node-role.kubernetes.io/master",
+						"effect": "NoSchedule"
+					}
+				],
+				"serviceAccountName": "weave-net",
+				"securityContext": {
```
Comment (luxas) [inline phase3/weave-net/weave-net-daemon-set.json]: I _think_ this was removed by WW some time ago

Comment (dims) [reply phase3/weave-net/weave-net-daemon-set.json]: this commit is just reformatting ... i am not adding or removing anything here

File: phase3/weave-net/weave-net-daemon-set.json (hunk 1 above)
Comment (dims) [reply phase3/weave-net/weave-net-daemon-set.json]: this commit is just reformatting ... i am not adding or removing anything here

**Comment by luxas** [comment]
@dims `git.io/weave-kube-1.6` should be the model I think

**Comment by dims** [comment]
ack will take a stab at it later as it looks like the problem we were seeing was not in the weave version itself

**Comment by bboreham** [comment]
If you are targeting Kubernetes 1.7 or 1.8 I would definitely add the `iptables` lock mount as well.
https://github.com/weaveworks/weave/blob/master/prog/weave-kube/weave-daemonset-k8s-1.7.yaml#L116-L118

**Comment by dims** [comment]
@luxas : @bboreham : done. just did the minimum (change version # and add the iptables lock mount)

**Pull request closed without merging**